### PR TITLE
Adopt CEL expressions for ignoring issues

### PR DESCRIPTION
**Pull request by ralphbean**
With intent to let teams ignore the parent link rule

### CODE REVIEW

**Comment by ralphbean** [comment]
We should generalize this into a decorator applied to all rules.

File: config/rhtapwatch.yaml [hunk 1]
```diff
@@ -7,7 +7,10 @@ team_automation:
     Epic:
       # collector: get_issues
       rules:
-        - check_parent_link
+        - rule: check_parent_link
+          kwargs:
+            # This is a CEL expression
+            ignore: '"orphan" in .labels'
```
Comment (amisstea) [inline config/rhtapwatch.yaml:13]: Is this label only utilized in the RHTAPWATCH project or should we apply this rule across other projects too?

Comment (ralphbean) [reply config/rhtapwatch.yaml:13]: It's only for you. You can pick the label you want and just set it here in the CEL expression. Your choice!

**Review by amisstea — APPROVED** [review]
LGTM

**Comment by Roming22** [comment]
I don't think it is a good idea to ignore the parent link rule. It means that there is work that is done that is not traceable back to a company priority. I.e. teams will spend time on things that are invisible to their (top) management, which then create blind spots.

**Pull request merged**